app/forms/update_user_email_form.rb

```diff
@@ -15,7 +15,7 @@ def initialize(user)
   end
 
   def submit(params)
-    email = params[:email].downcase
+    email = params[:email].downcase.strip
```

Comment:
I started writing this but realized this is probably not the right approach. Because the form passes the email around to other classes, a spec that tests that we are calling the utility class will still need to make sure the class returns the formatted email, so we would be essentially testing the class each time. For example:
```ruby
it 'calls EmailFormatter#downcase_and_strip' do
  formatter = instance_double(EmailFormatter)
  allow(EmailFormatter).to receive(:new).with('OLD@example.com ').and_return(formatter)
 
  expect(formatter).to receive(:downcase_and_strip).and_return('old@example.com')

  subject.submit(email: 'OLD@example.com ')
end
```

What I think we want to do is perform this normalization inside `FormEmailValidator`, ideally in the `valid_email` gem, much like the [phony_rails gem gives you the option to normalize the phone number before validating it](https://github.com/joost/phony_rails#normalization--model-usage). Calling `downcase` and `strip` on any user-entered email makes sense to me to do before validating it, and in every case where we are doing this manually, it is/should be via a Form Object that includes `FormEmailValidator`.